app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -255,6 +255,15 @@ def add_ep_merge_special_issue!
     update!(form_json: JSON.dump(form))
   end
 
+  def ep_merge_feature_enabled?
+    actor = OpenStruct.new({ flipper_id: user_uuid })
```

Comment:
Has the `Flipper` class changed? this seems like a strange pattern. before it accepted the `User` object, now just a struct with `flipper_id`?